packages/experiments/src/components/Card/Card.styles.ts

```diff
@@ -1,12 +1,33 @@
 import { ICardComponent, ICardStylesReturnType, ICardTokenReturnType } from './Card.types';
 import { getGlobalClassNames } from '../../Styling';
+import { Depths } from '@uifabric/fluent-theme';
 
 const GlobalClassNames = {
   root: 'ms-Card',
   stack: 'ms-Card-stack'
 };
 
-export const CardTokens: ICardComponent['tokens'] = (props, theme): ICardTokenReturnType => [];
+const baseTokens: ICardComponent['tokens'] = {
+  boxShadow: Depths.depth16,
+  padding: 12,
+  minWidth: '200px',
+  maxWidth: '250px'
+};
+
+const compactTokens: ICardComponent['tokens'] = {
+  minWidth: '300px',
+  maxWidth: '500px'
+};
+
+const clickableTokens: ICardComponent['tokens'] = {
+  boxShadowHovered: Depths.depth64
+};
+
+export const CardTokens: ICardComponent['tokens'] = (props, theme): ICardTokenReturnType => [
+  baseTokens,
+  props.compact && compactTokens,
+  props.onClick && clickableTokens
```

Comment:
So if it's not clickable we don't have a hover state change of the box-shadow?

Reply:
We have to follow up with design for that, but at the moment there is none 😄 .